### PR TITLE
Fixes/feedback 20190930

### PR DESCRIPTION
**Pull request by Nainterceptor**

### CODE REVIEW

File: components/ride-calendar/index.vue [hunk 1]
```diff
@@ -275,7 +276,7 @@ export default {
         ride.start = DateTime.fromISO(event.ride.start);
         ride.end = DateTime.fromISO(event.ride.end);
         ride.interval = Interval.fromDateTimes(ride.start, ride.end);
-        this.ride = ride;
+        this.ride = Object.assign(generateEmptyRide(), ride);
```
Comment (Rodrive) [inline components/ride-calendar/index.vue]: Why?

File: components/ride-calendar/modal.vue [hunk 2]
```diff
@@ -469,6 +503,7 @@ export default {
       return state.can(action);
     },
     copyToComments(content) {
+      console.log(content, this.ride.comments);
```
Comment (Rodrive) [inline components/ride-calendar/modal.vue]: No console.log, this will not pass test anyway I think

Comment (Rodrive) [reply components/ride-calendar/modal.vue]: Ok it seems that its only a lint warning but it shouldnt be there

Comment (Nainterceptor) [reply components/ride-calendar/modal.vue]: Removed !

**Pull request merged**